data/helpers.d/network

```diff
@@ -0,0 +1,9 @@
+# Check availability of a web path
+#
+# example: ynh_path_validity $domain$path
+#
+# usage: ynh_path_validity $domain_and_path
+# | arg: domain_and_path - complete path to check
+ynh_path_validity () {
```

Comment:
Actually it's not clear to me if this helper / function has side-effect or not. From the name of the helper, I would think it doesn't and that I need to run some sort of `ynh_url_path_register` if the path was found to be available. But from the code of `checkurl` it's not clear. Though it looks like if no app name is given, it doesn't have side-effects ?

Btw does this helper works in `$app` is not defined ?